packages/java/maven-plugin/src/main/java/dev/hilla/maven/EngineConfigureMojo.java

```diff
@@ -50,11 +50,8 @@ public final class EngineConfigureMojo extends AbstractMojo {
     public void execute() throws EngineConfigureMojoException {
         try {
             EngineConfiguration conf = new EngineConfiguration();
-            conf.setClassPath(Stream
-                    .of(project.getCompileClasspathElements(),
-                            project.getRuntimeClasspathElements())
-                    .flatMap(Collection::stream)
-                    .collect(Collectors.toCollection(LinkedHashSet::new)));
+            conf.setClassPath(
+                    new LinkedHashSet<>(project.getRuntimeClasspathElements()));
```

Comment:
Do we ignore compile-time classpath elements?

Reply:
Not really, according to [the source](https://github.com/apache/maven/blob/b82479bf111e4d0dc806d9be6d87f1fec4c2ba40/maven-core/src/main/java/org/apache/maven/project/MavenProject.java#L365), Maven does include compile-time artifacts there.

Reply:
That's... weird 🤨